packages/kbn-ui-theme/src/theme.ts

```diff
@@ -7,6 +7,8 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
+// TODO(tkajtoch): Add support for multiple themes
```

Comment:
This will be done in a follow-up PR adding the experimental Borealis theme as a dependency and importing its styles here